common/src/main/kotlin/orion/data/Task.kt

```diff
@@ -1 +1,24 @@
-package orion.data
\ No newline at end of file
+package orion.data
+import kotlinx.serialization.*
+
+// TODO - get this working 
+// 1 (highest) to 4 (lowest)
+// @Serializable
+// enum class Priority(val rgb: String)  { 
+//     P1("1"),
+//     P2("2"),
+//     P3("3"),
+//     P4("4") 
+// }
+
+// TODO - recurrence + subtasks 
+@Serializable
+data class Task(
+    val Id: String = "",
+    val CreatedAt: String = "",
+    var Description: String = "",
+    var Deadline: String = "",
```

Comment:
How do you feel about having this be `DueDate` instead of `Deadline`? All other task apps I've seen have used due date, and I tend to agree with them. Deadline sounds too "intense" to me